packages/clerk-js/src/ui/localization/makeLocalizable.tsx

```diff
@@ -71,11 +71,19 @@ export const useLocalizations = () => {
     if (!error || typeof error === 'string') {
       return t(error);
     }
-    const { code, message, longMessage } = error || {};
+    const { code, message, longMessage, meta } = error || {};
+    const { paramName = '' } = meta || {};
+
     if (!code) {
       return '';
     }
-    return t(localizationKeys(`unstable__errors.${code}` as any)) || longMessage || message;
+
+    return (
+      t(localizationKeys(`unstable__errors.${code}__${paramName}` as any)) ||
+      t(localizationKeys(`unstable__errors.${code}` as any)) ||
```

Comment:
❓ the reason behind the above 2 lines are to fallback to a more generic error (eg `unstable__errors.${code}`) or is to support the `paramName` being falsy?

Reply:
Both :)
If paramName is not found within`error.meta` or if it found but a specific translation is not found, then we fallback to a more generic translation or the `message` sent from FAPI.